scripts/os-stat.sh

```diff
@@ -7,7 +7,8 @@ set -e
 # set -x
 # set -v
 
-
+# Create mount
```

Comment:
??

Reply:
At first, we need to create the corresponding file in advance when you use mount. Later, the use of bind does not need to be created in advance. I forget to clean up the comments

Reply:
done